weblate/api/serializers.py

```diff
@@ -90,15 +90,35 @@ def __init__(self, *args, **kwargs):
 
 class LanguageSerializer(serializers.ModelSerializer):
     web_url = AbsoluteURLField(source="get_absolute_url", read_only=True)
+    plural = serializers.SerializerMethodField()
```

Comment:
This needs own serializer to support creating and updating plurals (in future, you don't have to implement that in this PR).

Reply:
Should I make it into a HyperLinkedIdentityField? Or just call the serializer with many=True?

Reply:
I think serializer with many is good here, these are probably not useful standalone to deserve own API endpoint.